content/300-guides/050-database/880-troubleshooting-orm/100-help-articles/425-nextjs-prisma-client-monorepo.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaTitle: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaDescription: 'Using Prisma Client in a Next.js project in a monorepo setup'
+---
+
+## Problem
+
+If you use Prisma Client in a [Next.js](https://nextjs.org/) application within a monorepo, you will likely run into an error such as the following:
+
+```
+Error: ENOENT: no such file or directory, open '/prisma-next-mono-reproduction/packages/service/.next/cache/webpack/client-development/schema.prisma'
+```
+
+Assume you have a monorepo with the following structure:
+
+```
+.
+├── packages/
+│   ├── prisma/
```

Comment:
Let's call this `db` - otherwise this can be confused with the normal `prisma` folder.
(The Prisma folder might actually be missing in the package, not sure if those usually include the schema file directly - @millsp?)